graviti/client/log.py

```diff
@@ -0,0 +1,268 @@
+# !/usr/bin/env python3
```

Comment:
Should this file be placed under `graviti/utility` folder? There is a circular import if log is placed here.